docs/developer-docs/integrations/sns/launching/launch-steps.md

```diff
@@ -69,30 +69,49 @@ sns deploy --network "${NETWORK}" --init-config-file "${CONFIG}" --save-to "sns_
 ### 5. Dapp developers submit an SNS proposal to handover control of their dapp to the SNS
 
```

Comment:
I think I just noticed what happened: 
the complete old description of 5. should be copy-pasted to Step 7. that currently does not have any description. 
Also, the description here should be adjusted - I 'll make a proposal in diffs below.
(I don't know how to make this suggestion as a diff, but basically step 7 should look like this afterwards:)

### 7. Proposal to start the decentralization swap
After the SNS canisters are deployed and the dapp's control is handed over to the SNS, an NNS proposal starts the swap.

Again, anyone who owns an NNS neuron with enough stake can submit this proposal.
Of course it is crucial to set the right parameters in this proposal.
You can also find an example how this command is used in the SNS local testing
[here](https://github.com/dfinity/sns-testing/blob/main/open_sns_sale.sh#L11-L26).
```
ic-admin   \
   --nns-url "${NETWORK_URL}" propose-to-open-sns-token-swap  \
   --min-participants 3  \
   --min-icp-e8s 5000000000  \
   --max-icp-e8s 50000000000  \
   --min-participant-icp-e8s 100000000  \
   --max-participant-icp-e8s 20000000000  \
   --swap-due-timestamp-seconds "${DEADLINE}"  \
   --sns-token-e8s 500000000000  \
   --target-swap-canister-id "${SNS_SWAP_ID}"  \
   --community-fund-investment-e8s 5000000000  \
   --neuron-basket-count 3  \
   --neuron-basket-dissolve-delay-interval-seconds 31536000  \
   --proposal-title "Decentralize this SNS"  \
   --summary "Decentralize this SNS"
```

Reply:
ah gotcha!

Reply:
done